webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -188,10 +211,6 @@ describe('ContributionForm.vue', () => {
         expect(wrapper.vm.form.content).toEqual(propsData.contribution.content)
       })
 
-      it('sets language equal to contribution language', () => {
-        expect(wrapper.vm.form.language).toEqual({ value: propsData.contribution.language })
-      })
```

Comment:
Why has this been removed?

Reply:
oversight, I think... sorry I should have caught that, have updated it